src/main/java/org/broadinstitute/hellbender/tools/walkers/sv/PrintAlleleCounts.java

```diff
@@ -0,0 +1,42 @@
+package org.broadinstitute.hellbender.tools.walkers.sv;
+
+import htsjdk.tribble.Feature;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.BetaFeature;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.programgroups.StructuralVariantDiscoveryProgramGroup;
+import org.broadinstitute.hellbender.engine.*;
+import org.broadinstitute.hellbender.tools.walkers.sv.PairedEndAndSplitReadEvidenceCollection.LocusDepth;
+
+@BetaFeature
+@CommandLineProgramProperties(
+        summary = "Prints allele counts.",
+        oneLineSummary = "Prints allele counts.",
+        programGroup = StructuralVariantDiscoveryProgramGroup.class
+)
+public class PrintAlleleCounts extends FeatureWalker<LocusDepth> {
```

Comment:
Can't remember, but I may have asked for this and wasn't clear about what I meant. Instead of this tool, can you add functionality to `PrintSVEvidence` to convert between locus depth bci and text formats? (consistent with the other evidence types)